exchanges/kraken.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+from exchanges import Exchange
+
+
+class Kraken(Exchange):
+
+    def __init__(self):
+        self.timeout = 5.0
+        self.url = 'https://api.kraken.com/0/'
+
+    def server_time(self):
+        """Queries the current server time (unix time stamp)"""
+
+        # Request the server time. This can raise an exception
+        # if there is a connection error or if the request times
+        # out. In addition, raise an exception if the status is not
+        # OK.
+        response = requests.get(self.url + 'public/Time', timeout=self.timeout)
```

Comment:
Maybe we should implement a _public_query method? as in: https://github.com/gagdiez/exchanges/blob/cassandra/exchanges/kraken/api.py. All the public requests to kraken follow the same format.